<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="0" failed="2" skipped="0">
  <reporter-output>
    <line>
      <![CDATA[Launching... https://opensource-demo.orangehrmlive.com/ application]]>
    </line>
    <line>
      <![CDATA[Login OrangeHRM web page ....]]>
    </line>
    <line>
      <![CDATA[Validate that you have logged in successfully]]>
    </line>
    <line>
      <![CDATA[Click Admin - Go to the admin page]]>
    </line>
    <line>
      <![CDATA[ Search username in field "Username"]]>
    </line>
  </reporter-output>
  <suite started-at="2021-07-09T02:42:38Z" name="Default suite" finished-at="2021-07-09T02:43:10Z" duration-ms="31847">
    <groups>
    </groups>
    <test started-at="2021-07-09T02:42:38Z" name="Default test" finished-at="2021-07-09T02:43:10Z" duration-ms="31847">
      <class name="com.opensource.admin.qa.TC004Admin_DeleteUser_POM">
        <test-method is-config="true" signature="beforeTest()[pri:0, instance:com.opensource.admin.qa.TC004Admin_DeleteUser_POM@5117dd67]" started-at="2021-07-09T02:42:38Z" name="beforeTest" finished-at="2021-07-09T02:42:41Z" duration-ms="2987" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method signature="TC003Admin_AddNewUser_POM_Script()[pri:0, instance:com.opensource.admin.qa.TC004Admin_DeleteUser_POM@5117dd67]" started-at="2021-07-09T02:42:41Z" name="TC003Admin_AddNewUser_POM_Script" finished-at="2021-07-09T02:42:41Z" duration-ms="20" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.opensource.admin.qa.TC003Admin_AddNewUser_POM.TC003Admin_AddNewUser_POM_Script(TC003Admin_AddNewUser_POM.java:57)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC003Admin_AddNewUser_POM_Script -->
        <test-method signature="TC004Admin_DeleteUser_POM_Script()[pri:0, instance:com.opensource.admin.qa.TC004Admin_DeleteUser_POM@5117dd67]" started-at="2021-07-09T02:42:41Z" name="TC004Admin_DeleteUser_POM_Script" finished-at="2021-07-09T02:43:10Z" duration-ms="28751" status="FAIL">
          <exception class="java.lang.IllegalArgumentException">
            <message>
              <![CDATA[Keys to send should be a not null CharSequence]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.IllegalArgumentException: Keys to send should be a not null CharSequence
at org.openqa.selenium.remote.RemoteWebElement.sendKeys(RemoteWebElement.java:97)
at com.opensource.base.SeleniumWrapper.type(SeleniumWrapper.java:121)
at com.opensource.admin.UserManagementPage.searchUser(UserManagementPage.java:127)
at com.opensource.admin.qa.TC004Admin_DeleteUser_POM.TC004Admin_DeleteUser_POM_Script(TC004Admin_DeleteUser_POM.java:62)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IllegalArgumentException -->
          <reporter-output>
            <line>
              <![CDATA[Launching... https://opensource-demo.orangehrmlive.com/ application]]>
            </line>
            <line>
              <![CDATA[Login OrangeHRM web page ....]]>
            </line>
            <line>
              <![CDATA[Validate that you have logged in successfully]]>
            </line>
            <line>
              <![CDATA[Click Admin - Go to the admin page]]>
            </line>
            <line>
              <![CDATA[ Search username in field "Username"]]>
            </line>
          </reporter-output>
        </test-method> <!-- TC004Admin_DeleteUser_POM_Script -->
        <test-method is-config="true" signature="afterTest()[pri:0, instance:com.opensource.admin.qa.TC004Admin_DeleteUser_POM@5117dd67]" started-at="2021-07-09T02:43:10Z" name="afterTest" finished-at="2021-07-09T02:43:10Z" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
      </class> <!-- com.opensource.admin.qa.TC004Admin_DeleteUser_POM -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
